ranger/data/scope.sh

```diff
@@ -92,7 +92,15 @@ handle_extension() {
         json)
             jq --color-output . "${FILE_PATH}" && exit 5
             python -m json.tool -- "${FILE_PATH}" && exit 5
-            ;;
+            ;; # Continue with next handler on failure
+	## Jupyter Notebook
+        ipynb)
+            ## Convert to HTML then preview as text conversion
```

Comment:
More weird indentation. Make sure to be using *spaces* not tabs.

Reply:
Done